AdminWebsite/AdminWebsite/ClientApp/src/app/dashboard/dashboard.component.html

```diff
@@ -1,3 +1,4 @@
+<app-unallocated-hearings></app-unallocated-hearings>
```

Comment:
Should this be behind the work allocation feature flag?

Reply:
Ah yeah, it wont show up in production as you need the vho team lead access, which no one will have. But you're right I should probably add the feature flag to the component just incase..